content/workshops/cascading-styles-deep.md

```diff
@@ -17,71 +21,120 @@ tags:
 ---
 
-A lot has changed in the nearly 30 years since CSS was first proposed,
-and new features are shipping at an unprecedented rate --
-cascade layers, container queries,
-the ':has()' selector, wide-gamut colors,
-subgrid, and so much more.
-
-Meanwhile, we're still getting used to
-custom properties and basic grids!
-Taken one at a time, the list can feel overwhelming,
-but these features all fit together to form a highly systemic language
-based on a radical foundation: _the cascade of styles_.
-Together, we’ll explore:
-
-## Selectors & the Cascade
-
-We’ll look at selectors,
-their specificity,
-nesting, and of course custom properties.
-
-- cascade origins and importance
-- cascade layers
-- specificity and source-order
-- :is(), :where(), and :has()
-- nesting & scope
-- preference & support queries
-- custom properties
-- naming & organizing conventions
-- progressive enhancement
-
-## Layout & Typography
-
-From modern CSS layout to functions,
-from aspect ratios to media & container queries.
-
-- display, flow, and context
-- writing modes
-- logical properties
-- flexbox, grid, and subgrid
-- comparison functions
-- aspect-ratios
-- scroll-snapping
-- media & container queries
-
-## CSS Colors
-
-Lastly, new color spaces & formats, accent-color and much more!
-
-- accent-color
-- color-scheme
-- new color spaces & formats
-- gradient color spaces
-- color-mix()
+It’s an exciting time, but the list can also feel overwhelming.
+Do I really need grids if I already know flexbox?
+What problem do layers actually solve?
+Should I rewrite my styles with each feature release,
+or stick with the tools that I know?
+
+If you ever feel lost in the CSS weeds,
+wrestling with  the cascade,
+or you just want to improve your workflow
+with modern, efficient, and maintainable stylesheets,
+then this workshop is for you!
+We'll approach the language as a cohesive system,
+learn new features along the way,
+and then put it all in practice
+with a resilient approach to CSS architecture.
+
+Join Miriam for a deep dive
+into what makes the language work,
+and how we can harness its power
+to **develop resilient and delightful experiences**
+that hold up across browsers, languages, and device interfaces.
+
+## What Will Attendees Learn In This Workshop?
+
+- Building maintainable CSS systems with progressive enhancement.
+- The entire cascade, why it exists,
+  and what all is involved (much more than specificity!)
+- Managing style conflicts with cascade layers, scope, and nesting.
+- Modern selectors like :is(), :where(), and :has().
+- Value resolution and validation, beyond cascading and inheritance.
+- Using custom properties to build adaptive systems.
+- The advantages of logical vs physical dimensions.
+- The layout flow with intrinsic and extrinsic sizing.
+- Size comparison functions like min(), max(), and clamp().
+- When to use flexbox vs grid for layout and alignment.
+- Fast and dynamic layouts using subgrid and container queries.
```

Comment:
I feel like most of these list items aren't actually sentences, so I wonder if including the `.` at the end of each line is necessary?